streamalert_cli/_infrastructure/_include.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = "~> 0.13.0"
+
+  required_providers {
+    aws = {
+      source  = "aws"
```

Comment:
Probably we need this `source  = "hashicorp/aws"`

Reply:
yeah you're right - updated